web/src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -107,8 +154,13 @@ export default {
     return {
       rootDirectory,
       location: rootDirectory,
-      loading: false,
       itemDownloads: {},
+      itemDeletes: {},
+      owners: [],
+      dialog: {
+        active: false,
+        name: '',
+      },
```

Comment:
This is another small thing, but I think in general we should discourage using objects to contain related information like this. The main reason being if down the line someone tries to watch/use/depend on the top-level `dialog` field, it will fail, because that field isn't reactive (only the inner items are). I'm curious what your opinions on this are @waxlamp and @dchiquito. All that said, if you have a strong enough reason to keep it I'm fine with that too.

Reply:
Ideally the `<v-dialog>` would be embedded in its own component, but I didn't want to do that here. I am fine with splatting these props into the top level, but I also am not too worried about someone trying to `watch` the object value, since it's sort of Vue-knowledge not to do that. As a self-counterpoint, we could avoid that error altogether by splatting these values.

So I'll wait to hear from @dchiquito but I am ok with unraveling this object in this case.

Reply:
I did not know that `dialog` would not be reactive, so make of that what you will. I prefer the object grouping syntactically, but I think I would prefer keeping all `data` reactive, if I had to have an opinion on this matter.

Vue3 pls

Reply:
> Vue3 pls

I agree with this plaintive cry for help and would like to subscribe to your newsletter.

See 1e0f47580e8d653130fbcbd5cf5a5c067321c97e.